modules/installation-guide/partials/proc_switching-between-external-and-internal-communication.adoc

```diff
@@ -2,29 +2,29 @@
 [id="switching-between-external-and-internal-communication_{context}"]
 = Switching between external and internal ways in inter-component communication
 
-The communication between {prod} components uses the internal cluster host name.
-In following situations, instead, use external {platforms-ingress} in inter-component communication:
+The communication between {prod} components uses the internal {orch-name} DNS names.
+In following situations, instead, use external {platforms-ingress} in inter-component communications:
 
 * Multi-cluster OpenShift environment.
-* The environment is restricting communication between namespaces.
+* The {orch-name} NetworkPolicies are restricting communication between namespaces.
 
 .Prerequisites
 
 * The `{orch-cli}` tool is available.
-* An instance of {prod-short} running in {platform-name}.
+* An instance of {prod-short} running in {orch-name}.
```

Comment:
Here we suppose that OpenShift is a Kubernetes.
But for CRW it will be OpenShift, since it's only supported

https://5fd8f6f28477ca23551786e5--eclipse-che-docs-pr.netlify.app/che-7/installation-guide/switching-between-external-and-internal-communication/

Reply:
`Kubernetes or OpenShift` doesn't makes sense to me. OpenShift is a Kubernetes distribution, hence if you say Kubernetes, OpenShift is already included. Upstream it should be `Kubernetes`, downstream `OpenShift`